### PR TITLE
Add contribution guide

### PR DESCRIPTION
**Pull request by aweell**
Add a new guides folder with the contribution guide

### CODE REVIEW

**Comment by tuentisre** [comment]
:robot: OpenAI

## High-level Summary
The pull request adds a contribution guide to the Mística Design System for external teams. The guide outlines the process for proposing, creating issues, updating or creating specifications, and moving from design to development. It also includes an index and approval criteria for proposals.

## Files Changed
| File | Summary |
| --- | --- |
| guides/contribution.md | Adds a contribution guide to the Mística Design System for external teams. The guide outlines the process for proposing, creating issues, updating or creating specifications, and moving from design to development. It also includes an index and approval criteria for proposals. |

---

### Chat with 🤖 OpenAI Bot (`@openai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@openai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

---

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:3]: :robot: OpenAI

The introduction is clear and concise. However, it would be helpful to add a brief summary of what the contribution guide entails.
```suggestion
# Contribution Guide

This guide outlines the process for proposing, creating issues, updating or creating specifications, and moving from design to development in the Mística Design System for external teams. It also includes an index and approval criteria for proposals.
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:8]: :robot: OpenAI

The paragraph provides a good overview of the purpose of the guide. However, it could be more specific about the benefits of contributing to the Mística Design System.
```suggestion
As an external team, you have the opportunity to contribute to the Mística Design System by sharing your ideas and expertise. Your contributions can help us improve the system and make it more useful for a wider range of users. This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:18]: :robot: OpenAI

The index provides a clear outline of the topics covered in the guide. However, it would be helpful to provide a brief description of each section.
```suggestion
## Index

- [Proposal](#proposals): Learn how to submit a proposal for a new component or feature.
- [Creation of an issue based on the proposal](#issues): Understand how to create an issue based on an approved proposal.
- [Updating or creating the specs needed](#specifications): Find out how to update or create specifications for components or features.
- [Development and final review](#from-design-to-development): Discover the steps involved in moving from design to development.
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:22]: :robot: OpenAI

The image provides a clear visual representation of the contribution journey. No changes needed.
```
![Contribution journey](../.github/resources/guides/contribution/contribution_journey.svg)
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:31]: :robot: OpenAI

The paragraph provides clear criteria for submitting a proposal. However, it would be helpful to provide more detail on what is expected in each criterion.
```suggestion
Before submitting a proposal, we encourage you to review our existing components and guidelines to ensure that your proposal aligns with our design principles. Additionally, please ensure that there is no other discussion already covering the same topic. You can find all of our design guidelines and documentation on [Telefónica's brand factory](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/get-started/what-is-mistica). Please also ensure that the proposed change is not covered by the existing [Component specs](https://www.figma.com/files/1125734703130062955/project/27955986/Component-Specs?fuid=1111936175780412673) and that it is technically viable to include the proposal in your product (A validated PRD, or confirmation that backend can serve this information).
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:41]: :robot: OpenAI

The table provides clear approval criteria for proposals. No changes needed.
```
| Approval Criteria | Description                                                                                                                                                                                                                  |
| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Global            | The proposal should be applicable to a broad range of use cases and scenarios, rather than being specific to a particular product or project. This ensures that the component is useful to a wider range of users and teams. |
| Purpose           | The purpose of the proposed change needs to be well-explained. What the component is designed to solve, the benefits it provides, and any potential drawbacks or limitations.                                                |
| Figma link        | You should provide a Figma link with examples of how the component could be used in different contexts. This helps us visualize the proposed change and understand its potential impact on the overall design system.        |
```

<!-- This is an auto-generated comment by OpenAI -->

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:47]: :robot: OpenAI

The paragraph provides helpful information for validating a proposal. No changes needed.
```
Other information could help to validate the proposal earlier:

- Research data.
- Examples of current usage.
- Examples of where the proposal could solve a problem more efficiently.
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:53]: :robot: OpenAI

The paragraph provides clear information on what happens after a proposal is approved. No changes needed.
```
When a proposal is approved an issue will be create from the discussion. The issues can either be assigned to the design core team or any external team, depending on different factors, like workload, complexity, impact. The definition of the assignee will be discussed between the design core team and the teams involved in the proposal.
```

File: guides/contribution.md
Comment (tuentisre) [inline guides/contribution.md:62]: :robot: OpenAI

The paragraph provides clear information on how to track progress when the design core team is assigned to update the specs based on the proposal. No changes needed.
```
If design core has been defined as the asignee for updating the specs based on the proposal you can track the progress from:

- [Our milestones](https://github.com/Tele

<!-- This is an auto-generated comment by OpenAI -->

**Review by yceballost — APPROVED** [review]
clarinete 🤩

**Pull request merged**
